chapters/comparing-models.qmd

```diff
@@ -938,75 +996,67 @@ To adapt this idea to a single validation set, we treat the validation set as ou
 
-The bootstrap approach was applied to the three models for the food delivery time data.  @fig-bootstrap-validation-delivery illustrates the 90% bootstrap confidence intervals for each of the models.  Notice that these intervals are narrow.  This is because the number of validation set samples is large (n = `r nrow(delivery_test)`).
+The bootstrap approach was applied to the three models for the forestation classification data.  @fig-bootstrap-validation-forested illustrates the 90% bootstrap confidence intervals of accuracy for each of the models.  Notice that these intervals are narrow.  This is because the number of validation set samples is large (n = `r nrow(forested_test)`).
 
 ```{r}
-#| label: fig-bootstrap-validation-delivery
+#| label: fig-bootstrap-validation-forested
 #| echo: false
 #| warning: false
 #| cache: true
 #| fig-width: 6
 #| fig-height: 3.5
 #| fig-align: "center"
 #| out-width: "70%"
-#| fig-cap: 90% bootstrap confidence intervals of the validation set MAE for each model.  The circle represent the MAE estimate of the validation set for each model.
-
+#| fig-cap: 90% bootstrap confidence intervals of the validation set accuracy for each model.  The circle represent the accuracy estimate of the validation set for each model.
 # Get validation set performance and corresponding bootstrap CI
-rf_validation_res <-
-  rf_spec %>%
-  last_fit(time_to_delivery ~ ., split = delivery_split, metrics = reg_metrics)
-
-rf_validation_boot <- int_pctl(rf_validation_res, alpha = 0.90)
-
-cb_spec <-
-  cubist_rules(committees = 5)
-
-cb_validation_res <-
-  cb_spec %>%
-  last_fit(time_to_delivery ~ ., split = delivery_split, metrics = reg_metrics)
-
-cb_validation_boot <- int_pctl(cb_validation_res, alpha = 0.90)
-
-nnet_validation_res <-
-  nnet_wflow %>%
-  last_fit(time_to_delivery ~ ., split = delivery_split, metrics = reg_metrics)
-
-nnet_validation_boot <- int_pctl(nnet_validation_res, alpha = 0.90)
-
-validation_boot_mae <-
-  bind_rows(
-    rf_validation_boot %>% mutate(Model = "Random Forest"),
-    cb_validation_boot %>% mutate(Model = "Cubist"),
-    nnet_validation_boot %>% mutate(Model = "Neural Network")
-  ) %>%
-  dplyr::filter(.metric == "mae") %>%
-  mutate(
-    Model = factor(
-      Model,
-      levels = c("Random Forest", "Neural Network", "Cubist")
-    )
-  )
-
-ggplot(validation_boot_mae, aes(x = Model, y = .estimate)) +
-  geom_point(color = "black", shape = 1) +
-  geom_errorbar(
-    data = validation_boot_mae,
-    aes(ymin = .lower, ymax = .upper),
-    color = "black",
-    width = 0.05
-  ) +
-  ylab("MAE")
+glmnet_validation_res <-
+    glmnet_final_wflow %>%
+    last_fit(class ~ ., split = forested_split, metrics = class_metrics)
+
+glmnet_validation_boot <- int_pctl(glmnet_validation_res, alpha = 0.10)
+
+
+c50_validation_res <-
```

Comment:
I'd like to make a function that also does the traditional CI for a proportion and show how the bootstrap mirrors that. This will give them some confidence that it actually works